src/views/datasets/osom/summary.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
-      The Ocean State Ocean Model (OSOM) is an application of the Regional Ocean
-      Modeling System spanning the Rhode Island waterways, including
-      Narragansett Bay, Mt. Hope Bay, larger rivers, and the Block Island Shelf
-      circulation from Long Island to Nantucket. Data is available by the year
-      at 1.5 hour increments. As the model covers the entire Narragansett Bay,
-      data is always available at all buoy locations.
+    The Ocean State Ocean Model (OSOM), developed in a collaboration between
```

Comment:
could double check you ran `npm run lint`? A bit of the indenting/line breaks looks a bit off to me (but not very off, so it also might be right 🤷)